allensdk/core/typing.py

```diff
@@ -1,8 +1,12 @@
-from typing import _Protocol
+import sys
+if sys.version_info.minor <= 7:  
```

Comment:
Thanks Doug! I think a try-except block is more pythonic.

e.g.
```
try:
    from typing import _Protocol as Protocol
except ImportError:
    from typing import Protocol
```

Reply:
Technically we are not supporting 3.8 yet. I'm not sure if this is the only thing that will need to be updated. #1115

Reply:
I changed the if-statement to a try/except as suggested by @kschelonka